backend/requirements.txt

```diff
@@ -18,3 +18,7 @@ typing_extensions==4.12.2
 drf-spectacular==0.27.2
 djangorestframework==3.15.2
 djangorestframework-simplejwt==5.3.1
+
+# code coverage
+coverage==6.2
+coveralls==3.3.1
```

Comment:
please use latest coverage and coveralls version

Reply:
noted, will do